src/context/SyntheticsStateContext/selectors/positionsSelectors.ts

```diff
@@ -0,0 +1,23 @@
+import { createSelector } from "../utils";
+import { selectPositionsInfoData } from "./globalSelectors";
+import { selectTradeboxMarketsOrderMap } from "./tradeboxSelectors";
+import { bigintToNumber } from "lib/numbers";
+
+export const selectPositionsInfoDataSortedByMarket = createSelector((q) => {
+  const positionsInfoData = q(selectPositionsInfoData);
+  const marketsOrder = q(selectTradeboxMarketsOrderMap);
+
+  const positions = Object.values(positionsInfoData || {});
+
+  const sortedPositions = positions.sort((a, b) => {
+    const bMarketIdx = marketsOrder[a.indexToken.symbol];
+    const aMarketIdx = marketsOrder[b.indexToken.symbol];
+
+    if (aMarketIdx === bMarketIdx) {
+      return bigintToNumber(b.sizeInUsd - a.sizeInUsd, 1);
```

Comment:
hmmmm,
are not we afraid that this might overflow ?
maybe just A - B > 0n ? 1 : -1 ?